Applications/Spire/Include/Spire/BookView/BookViewHighlightPropertiesWidget.hpp

```diff
@@ -32,21 +34,25 @@ namespace Spire {
 
     private:
       QListWidget* m_markets_list_widget;
-      Checkbox* m_highlight_none_checkbox;
-      Checkbox* m_highlight_top_level_checkbox;
-      Checkbox* m_highlight_all_levels_checkbox;
+      std::shared_ptr<AssociativeValueModel<QString>>
```

Comment:
I wasn't sure what other type to use because the dialog isn't interested in the value associated with the CheckBox, only whether it's checked or unchecked. I could define a private enum with a value for each CheckBox to give it more meaning.